src/fieldlist_CMIP.jsonc

```diff
@@ -8,7 +8,7 @@
     "lon" : {"axis" : "X", "MDTF_envvar" : "lon_coord"},
```

Comment:
You don't need to make edits to the fieldlist files:
- Adding variable names for the clear-sky fluxes is our responsibility (issue #134).
- Coordinate names can be set to whatever you like via the new settings format.